### PR TITLE
Add test for lme

### PR DESCRIPTION
**Pull request by BenjaminHsieh**
Added test for lme functions, please check

@boyinggong I noticed you included a bonferroni correction on the prop of anova that are significant (anovaStat) but not for the propotion of beta -pvalues (calcSigProp). Should we add the correction?

### CODE REVIEW

**Comment by BenjaminHsieh** [comment]
Added test for smoothing function, please check

File: code/scripts/lme_script.py [hunk 1]
```diff
@@ -75,7 +66,7 @@
     sig_gain_prop[i-1], sig_loss_prop[i-1] = calcSigProp(beta, sig_level)
     write=pathtofolder + 'ds005/sub0'+str(i).zfill(2)+'/model/model001/onsets/sub0'+str(i).zfill(2)+'_lme_beta.txt'
     np.savetxt(write, beta)
-    anov_test = calcAnov(data_full, run_group)
+    anov_test = calcAnov(data_full, run_group, thrshd)
```
Comment (briankleinqiu) [inline code/scripts/lme_script.py:69]: should it be thrshd = None?

Looks good to me but I didn't run it yet

Edit: jk just saw thrshd = 400

**Comment by BenjaminHsieh** [comment]
is this good ?

**Comment by BenjaminHsieh** [comment]
Lets mege this

**Pull request merged**